diff -Nru fotema-1.15.1.old/Cargo.toml fotema-1.15.1/Cargo.toml
--- fotema-1.15.1.old/Cargo.toml	2024-10-16 01:16:24.426047021 +0200
+++ fotema-1.15.1/Cargo.toml	2024-10-16 01:16:24.594044425 +0200
@@ -16,6 +16,7 @@
 lto = "thin"
 
 [dependencies]
+glib-sys = { version = "0.20.4", features = ["v2_78"] }
 gettext-rs = { version = "0.7", features = ["gettext-system"] }
 tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.18", features = ["chrono", "env-filter"] }
