diff -Nru fotema-1.14.2.old/core/src/machine_learning/face_extractor.rs fotema-1.14.2/core/src/machine_learning/face_extractor.rs
--- fotema-1.14.2.old/core/src/machine_learning/face_extractor.rs	2024-09-28 01:16:35.349484671 +0200
+++ fotema-1.14.2/core/src/machine_learning/face_extractor.rs	2024-09-28 01:16:35.417483714 +0200
@@ -353,7 +353,7 @@
         let file = gio::File::for_path(source_path);
 
         let mut loader = glycin::Loader::new(file);
-        loader.sandbox_selector(glycin::SandboxSelector::FlatpakSpawn);
+        loader.sandbox_selector(glycin::SandboxSelector::Bwrap);
         let image = loader.load().await?;
         let frame = image.next_frame().await?;
         let bytes = frame.texture().save_to_png_bytes();
diff -Nru fotema-1.14.2.old/core/src/photo/thumbnail.rs fotema-1.14.2/core/src/photo/thumbnail.rs
--- fotema-1.14.2.old/core/src/photo/thumbnail.rs	2024-09-28 01:16:35.353484615 +0200
+++ fotema-1.14.2/core/src/photo/thumbnail.rs	2024-09-28 01:16:35.413483770 +0200
@@ -121,7 +121,7 @@
         let file = gio::File::for_path(source_path);
 
         let mut loader = glycin::Loader::new(file);
-        loader.sandbox_selector(glycin::SandboxSelector::FlatpakSpawn);
+        loader.sandbox_selector(glycin::SandboxSelector::Bwrap);
 
         let image = loader.load().await?;
 
diff -Nru fotema-1.14.2.old/src/app/components/viewer/view_one.rs fotema-1.14.2/src/app/components/viewer/view_one.rs
--- fotema-1.14.2.old/src/app/components/viewer/view_one.rs	2024-09-28 01:16:35.385484164 +0200
+++ fotema-1.14.2/src/app/components/viewer/view_one.rs	2024-09-28 01:16:35.425483601 +0200
@@ -351,7 +351,7 @@
                     let file = gio::File::for_path(visual_path);
 
                     let mut loader = glycin::Loader::new(file);
-                    loader.sandbox_selector(glycin::SandboxSelector::FlatpakSpawn);
+                    loader.sandbox_selector(glycin::SandboxSelector::Bwrap);
                     loader.apply_transformations(false);
 
                     let image = loader.load().await;
