diff -Nru fotema-1.14.4.old/core/src/photo/scanner.rs fotema-1.14.4/core/src/photo/scanner.rs
--- fotema-1.14.4.old/core/src/photo/scanner.rs	2024-10-26 01:24:09.033315444 +0200
+++ fotema-1.14.4/core/src/photo/scanner.rs	2024-10-26 01:24:10.285297071 +0200
@@ -45,6 +45,7 @@
         ];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
diff -Nru fotema-1.14.4.old/core/src/video/scanner.rs fotema-1.14.4/core/src/video/scanner.rs
--- fotema-1.14.4.old/core/src/video/scanner.rs	2024-10-26 01:24:09.037315386 +0200
+++ fotema-1.14.4/core/src/video/scanner.rs	2024-10-26 01:24:10.289297013 +0200
@@ -35,6 +35,7 @@
         let suffixes = [String::from("mov"), String::from("mp4")];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
