diff -Nru fotema-1.14.1.old/data/app.fotema.Fotema.desktop.in.in fotema-1.14.1/data/app.fotema.Fotema.desktop.in.in
--- fotema-1.14.1.old/data/app.fotema.Fotema.desktop.in.in	2024-09-15 01:14:02.070363555 +0200
+++ fotema-1.14.1/data/app.fotema.Fotema.desktop.in.in	2024-09-15 01:14:02.346359620 +0200
@@ -8,7 +8,8 @@
 # Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
 Keywords=Gnome;GTK;Pictures;Photos;Photography;Viewer;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
-Icon=@icon@
+Icon=/usr/share/icons/hicolor/scalable/apps/app.fotema.Fotema.svg
 StartupNotify=true
 # Translators: Do NOT translate or transliterate this text (these are enum types)!
 X-Purism-FormFactor=Workstation;Mobile;
+StartupWMClass=fotema
