diff -Nru fotema-1.15.2.old/core/src/photo/scanner.rs fotema-1.15.2/core/src/photo/scanner.rs
--- fotema-1.15.2.old/core/src/photo/scanner.rs	2024-10-18 01:19:34.973579473 +0200
+++ fotema-1.15.2/core/src/photo/scanner.rs	2024-10-18 01:19:36.845551677 +0200
@@ -45,6 +45,7 @@
         ];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
diff -Nru fotema-1.15.2.old/core/src/video/scanner.rs fotema-1.15.2/core/src/video/scanner.rs
--- fotema-1.15.2.old/core/src/video/scanner.rs	2024-10-18 01:19:34.985579295 +0200
+++ fotema-1.15.2/core/src/video/scanner.rs	2024-10-18 01:19:36.849551618 +0200
@@ -35,6 +35,7 @@
         let suffixes = [String::from("mov"), String::from("mp4")];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
