diff -Nru fotema-1.19.1.old/data/app.fotema.Fotema.desktop.in.in fotema-1.19.1/data/app.fotema.Fotema.desktop.in.in
--- fotema-1.19.1.old/data/app.fotema.Fotema.desktop.in.in	2025-02-04 05:44:52.440332708 +0100
+++ fotema-1.19.1/data/app.fotema.Fotema.desktop.in.in	2025-02-04 05:44:52.500331985 +0100
@@ -8,7 +8,7 @@
 # Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
 Keywords=Gnome;GTK;Pictures;Photos;Photography;Viewer;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
-Icon=@icon@
+Icon=/usr/share/icons/hicolor/scalable/apps/app.fotema.Fotema.svg
 StartupNotify=true
 # Translators: Do NOT translate or transliterate this text (these are enum types)!
 X-Purism-FormFactor=Workstation;Mobile;
