diff -Nru fotema-1.15.0.old/core/src/photo/scanner.rs fotema-1.15.0/core/src/photo/scanner.rs
--- fotema-1.15.0.old/core/src/photo/scanner.rs	2024-10-15 01:20:15.090506847 +0200
+++ fotema-1.15.0/core/src/photo/scanner.rs	2024-10-15 01:20:16.890479894 +0200
@@ -45,6 +45,7 @@
         ];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
diff -Nru fotema-1.15.0.old/core/src/video/scanner.rs fotema-1.15.0/core/src/video/scanner.rs
--- fotema-1.15.0.old/core/src/video/scanner.rs	2024-10-15 01:20:15.102506668 +0200
+++ fotema-1.15.0/core/src/video/scanner.rs	2024-10-15 01:20:16.894479834 +0200
@@ -35,6 +35,7 @@
         let suffixes = [String::from("mov"), String::from("mp4")];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
