diff -Nru fotema-1.14.4.old/core/src/photo/scanner.rs fotema-1.14.4/core/src/photo/scanner.rs
--- fotema-1.14.4.old/core/src/photo/scanner.rs	2024-10-30 01:24:02.918129876 +0100
+++ fotema-1.14.4/core/src/photo/scanner.rs	2024-10-30 01:24:05.210095323 +0100
@@ -45,6 +45,7 @@
         ];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
diff -Nru fotema-1.14.4.old/core/src/video/scanner.rs fotema-1.14.4/core/src/video/scanner.rs
--- fotema-1.14.4.old/core/src/video/scanner.rs	2024-10-30 01:24:02.926129755 +0100
+++ fotema-1.14.4/core/src/video/scanner.rs	2024-10-30 01:24:05.218095202 +0100
@@ -35,6 +35,7 @@
         let suffixes = [String::from("mov"), String::from("mp4")];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
