diff -Nru fotema-1.19.0.old/core/src/photo/scanner.rs fotema-1.19.0/core/src/photo/scanner.rs
--- fotema-1.19.0.old/core/src/photo/scanner.rs	2025-01-24 01:25:20.106882530 +0100
+++ fotema-1.19.0/core/src/photo/scanner.rs	2025-01-24 01:25:20.330878745 +0100
@@ -46,6 +46,7 @@
         ];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
diff -Nru fotema-1.19.0.old/core/src/video/scanner.rs fotema-1.19.0/core/src/video/scanner.rs
--- fotema-1.19.0.old/core/src/video/scanner.rs	2025-01-24 01:25:20.106882530 +0100
+++ fotema-1.19.0/core/src/video/scanner.rs	2025-01-24 01:25:20.334878678 +0100
@@ -35,6 +35,7 @@
         let suffixes = [String::from("mov"), String::from("mp4")];
 
         WalkDir::new(&self.scan_base)
+            .follow_links(true)
             .into_iter()
             .inspect(|x| {
                 let _ = x
