diff -Nru amiberry-7.0.1.old/packaging/linux/Amiberry.desktop amiberry-7.0.1/packaging/linux/Amiberry.desktop
--- amiberry-7.0.1.old/packaging/linux/Amiberry.desktop	2025-01-28 01:32:52.464807984 +0100
+++ amiberry-7.0.1/packaging/linux/Amiberry.desktop	2025-01-28 01:32:55.236761582 +0100
@@ -31,3 +31,5 @@
 
 # Describes the encoding for the desktop entry
 Encoding=UTF-8
+
+StartupWMClass = amiberry
